src/wasm.h

```diff
@@ -659,9 +660,6 @@ class Expression {
     assert(int(_id) == int(T::SpecificId));
     return (const T*)this;
   }
-
-  // Print the expression to stderr. Meant for use while debugging.
-  void dump();
```

Comment:
`dump` method can be useful in gdb; `<<` operator is hard to use in gdb, so `dump` might be good to keep.

Reply:
Oh, I didn't think about that... I restored `dump` now.